exercises/concept/maze-maker/src/MazeMakerSupport.elm

```diff
@@ -0,0 +1,62 @@
+module MazeMakerSupport exposing (..)
```

Comment:
I don't think these are used in the exemplar / solution, so maybe its better for these to go in the test module? Maybe I'm missing something ...

Reply:
I do mention `MazeMakerSupport.mazeDepth` in the instructions, so I thought it would be nice to show it. 
Also, if I put those functions in the test file, the students will not be able to look at the definition at the same time as their failing test.
In general, I find that I almost never look at the test file since Exercism shows me the test snippet, so maybe this is more discoverable? I don't really know 🤷‍♂️ 

What do you think?

Reply:
I don't think its a big deal either way, we could just wait for any feedback?

The only downside I see is that students might think they have to use it, being as its provided. I have no idea if anyone will think this though ...